lib/bpk-component-calendar/README.md

```diff
@@ -64,6 +64,47 @@ class App extends Component {
 }
 ```
 
+### Footer view
+
+NOTE: this is only supported for Android, it will have no effect on iOS.
+
+The Android calendar supports the notion of a footer view for each month, we provide a `highlightedDaysFooterView`
+implementation to show a list of highlighted days for each month:
+
+```js
+import React from 'react';
+import BpkCalendar, {
+  SELECTION_TYPES,
+  highlightedDaysFooterView,
+  colorBucketHighlight,
+  DateMatchers,
+} from 'backpack-react-native/bpk-component-calendar';
+
+const App = () => (
+  <BpkCalendar
+    locale={'en-gb'}
+    selectionType={SELECTION_TYPES.range}
+    minDate={Date.UTC(2019, 0, 2)}
+    maxDate={Date.UTC(2019, 11, 31)}
+    // You can optionally use a highlight color bucket to highlight a day cell in the calendar,
+    // using the same style used by `highlightedDaysFooterView`
+    colorBuckets={[
+      colorBucketHighlight(
+        DateMatchers.any(Date.UTC(2019, 0, 10)),
+      ),
+    ]}
+    footerView={highlightedDaysFooterView({
```

Comment:
Discussed on Slack but just for completeness, I think `androidFooterView` or `android_FooterView` helps to self document this prop